dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+pytest
```

Comment:
I find it's always useful to hint at the version you were using like `pytest>=X.X.X` because with open ended requirements, it's always possible that something will break your code in the future (probably for someone else :)). If you at least have a hint to which version *used to work* then someone can deduce the problem easier.